docs/en/patterns/table-mobile-card.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Table mobile card
+table_of_contents: true
+---
+
+# Table mobile card
+
+Applying the class `.p-table--mobile-card` will give any table a new responsive card view when on smaller screens. Each cell will require
+an `[aria-label]` to describe the cell on a mobile screen. We use the content to create a pseudo element and keep it line with the content.
+
+The `<thead>` element is completely hidden from  view on a smaller screen and if the table holds a `.p-contextual-menu` pattern all the children elements will be visible and be interactive.
+
+<a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/table-mobile-card/"
```

Comment:
The example URL is: `examples/patterns/table-mobile-card/table-mobile-card`. You should only create a new directory if you have more then one example.